libs/content/workspace/src/workspace.rs

```diff
@@ -60,6 +60,20 @@ pub enum WsMsg {
     Dirtyness(DirtynessMsg),
 }
 
+pub struct FileLoadedMsg {
+    id: Uuid,
+    is_new_file: bool,
+    tab_created: bool,
+    content: Result<(Option<DocumentHmac>, DecryptedDocument), TabFailure>,
+}
+
+pub struct SaveResult {
```

Comment:
I thought this cleanup would be helpful for this; it wasn't, but I still want it